src/main/java/com/stories/API/ServletInitializer.java

```diff
@@ -0,0 +1,13 @@
+package com.stories.API;
+
+import org.springframework.boot.builder.SpringApplicationBuilder;
+import org.springframework.boot.web.servlet.support.SpringBootServletInitializer;
+
+public class ServletInitializer extends SpringBootServletInitializer {
+
+	@Override
+	protected SpringApplicationBuilder configure(SpringApplicationBuilder application) {
+		return application.sources(ApiApplication.class);
+	}
+
+}
```

Comment:
Which is the purpose of this initializer?

Reply:
The web server is initialized and returned to the App.